packages/blade/src/components/ChatBubble/Rotate.web.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { LazyMotion, domAnimation, m } from 'framer-motion';
+import { castWebType } from '~utils';
+import { useTheme } from '~components/BladeProvider';
+import { msToSeconds } from '~utils/msToSeconds';
+import { cssBezierToArray } from '~utils/cssBezierToArray';
+
+const Rotate = ({
+  children,
+  animate,
+}: {
+  children: React.ReactElement;
+  animate?: boolean;
+}): React.ReactElement => {
+  const { theme } = useTheme();
+
+  return (
+    <LazyMotion features={domAnimation}>
```

Comment:
@saurabhdaware now that blade internally uses motion.div, do we need to put this LazyMotion on top level blade provider or can we skip and expect consumers to add motion providers?

Reply:
https://github.com/razorpay/blade/blob/master/packages/blade/docs/migration-docs/upgrade-v12.md

LazyMotion itself is added on consumer-level only so ideally we should remove it from here. I've already added it to our Storybook so our examples should work fine without this

Reply:
removed lazyMotion